crates/store/re_protos/proto/rerun/v1alpha1/cloud.proto

```diff
@@ -515,8 +515,6 @@ message QueryRange {
 }
 
 message GetChunksRequest {
-  rerun.common.v1alpha1.EntryId dataset_id = 1;
-
```

Comment:
Please err on the verbose side with docstring concerning all of the implicit API contracts (here, headers).

In an ideal world, both the `rcp` and the `message *Request` docstring would repeat the full thing about required/optional headers. This is obviously too much, but I think we could go with a quick reference to the "standard dataset headers"